src/nunit.analyzers/ClassicModelAssertUsage/ClassicModelAssertUsageCodeFix.cs

```diff
@@ -55,6 +55,12 @@ public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
             // Now, replace the arguments.
             List<ArgumentSyntax> arguments = invocationNode.ArgumentList.Arguments.ToList();
 
+            // Remove null message to avoid ambiguous calls.
+            if (arguments.Count == 3 && arguments[2].Expression.Kind() == SyntaxKind.NullLiteralExpression)
+            {
+                arguments.RemoveAt(2);
+            }
+
```

Comment:
This needs to move after the call to `UpdateArguments` below so that it also works if there is a tolerance value specified.